### PR TITLE
Updated Vertical Method to force flex and align-items on parent

### PR DESCRIPTION
**Pull request by lyndonmoore**
Found an issue when trying to use the vertical textfit method on a template. Through testing discovered that if the parent container is not set to flex and does not have the align items property set the textfit will not work correctly. Updated to enforce these styles automatically.

### CODE REVIEW

File: src/textfit.js [hunk 1]
```diff
@@ -19,6 +21,13 @@ TextFit.prototype.initStyles = function() {
     elem.style['display'] = self.fitMode === "horizontal" ? "inline-block" : "initial";
     // Store the Initial Font Size in a Data-Attribute to compare with later
     elem.dataset.fontSize = window.getComputedStyle(elem).fontSize;
+
+    // If we are using vertical we need to ensure the parent container styles are correctly set also
+    if(self.fitMode == "vertical") {
```
Comment (nmqanh) [inline src/textfit.js]: this should be === 'vertical' 
In JS, we should use `===` and use single quote rather than double quote.

Comment (nmqanh) [reply src/textfit.js]: also please replace double quote by single quote in other places

**Comment by lyndonmoore** [comment]
Updated with changes.

**Comment by nmqanh** [comment]
thanks a lot mate

**Pull request merged**